server/src/weaverbird/pipeline/pipeline.py

```diff
@@ -193,4 +265,5 @@ def render(self, variables: dict[str, Any], renderer) -> Pipeline:
             step.render(variables, renderer) if hasattr(step, "render") else step  # type: ignore
             for step in self.steps
         ]
-        return Pipeline(steps=steps_rendered)
+        # We clean __VOID__ values from filter steps conditions.
+        return Pipeline(steps=remove_void_conditions_from_filter_steps(steps_rendered))
```

Comment:
This is a breaking change, and I believe it goes beyond the scope of just rendering the variables

Reply:
sure you're right, it's breaking changes and is it not what we want at the end of the day ?
render variables inside the pipeline but make sure to remove also __VOID __ elements... or do you suggest me to create another method like `render_with_no_void` for this class to have that feature ? 🤔